apps/docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
+
+```
+$ yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+Using SSH:
+
+```
+$ USE_SSH=true yarn deploy
+```
+
+Not using SSH:
+
+```
+$ GIT_USER=<Your GitHub username> yarn deploy
+```
```

Comment:
The `yarn` stuff probably doesn't belong here. Are we planning to host this with GitHub pages?

Reply:
Yes, that was the plan. Still might have to diddle a bit with the config to get it hosted correctly. Will look into it

Reply:
Also removed the yarn stuff and added some more related commands for us :)